cleanlab_studio/studio/studio.py

```diff
@@ -144,3 +144,60 @@ def apply_corrections(self, cleanset_id: str, dataset: Any) -> Any:
             corrected_ds[label_column] = joined_ds["__cleanlab_final_label"]
             corrected_ds = corrected_ds[joined_ds["action"] != "exclude"]
             return corrected_ds
+
+    def create_project(
+        self,
+        dataset_id: str,
+        project_name: str,
+        *,
+        tasktype: str = "multi-class",
+        modality: Optional[str] = None,
+        modeltype: str = "regular",
+        label_column: Optional[str] = None,
+        feature_columns: Optional[List[str]] = None,
+        text_column: Optional[str] = None,
+    ) -> str:
+        dataset_details = api.get_dataset_details(self._api_key, dataset_id)
+
+        if modality is None:
+            print("Modality not supplied. Using dataset modality")
+            modality = dataset_details["modality"]
+
+        if label_column is not None:
+            if label_column not in dataset_details["label_columns"]:
+                raise ValueError(
+                    f"Invalid label column: {label_column}. Label column must have categorical feature type"
+                )
+        else:
+            label_column = str(dataset_details["label_column_guess"])
+            print(f"Label column not supplied. Using best guess {label_column}")
+
+        if feature_columns is not None and modality != "tabular":
+            print("Project modality is not tabular. Ignoring feature columns")
```

Comment:
as a general principle, we should be failing on these types of things. if a user supplies an argument, they expect it to be used. if the argument can't be used, we should fail and alert the user. same principle applies for the text column conditional below